src/utils/rootkeypackage_utils/inc/aduc/rootkeypackage_utils.h

```diff
@@ -10,9 +10,19 @@
 #define ROOTKEYPACKAGE_UTILS_H
 
 #include "aduc/rootkeypackage_types.h"
+#include <aduc/c_utils.h>
 #include <aduc/result.h>
 
+EXTERN_C_BEGIN
+
 ADUC_Result ADUC_RootKeyPackageUtils_Parse(const char* jsonString, ADUC_RootKeyPackage* outRootKeyPackage);
+
+void ADUC_RootKeyPackageUtils_DisabledRootKeys_Cleanup(ADUC_RootKeyPackage* rootKeyPackage);
```

Comment:
Cleanup implies the deletion of something on disk in my head. These are closer to Destroy or Delete functions

Reply:
Fair enough. I've changed it to _Destroy